WooCommerce/Classes/Extensions/Error+Woo.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+extension Error {
+    /// If a localized description is available, use it for the error alert.
+    /// Otherwise, use the provided fallback string.
+    func prepareErrorMessage(fallback: String) -> String {
```

Comment:
Nice extension method. Kindly consider adding a unit test for this.

Reply:
Good point! I added a test and found that `localizedDescription` is never nil or empty, so I removed the extension and switched to using `localizedDescription` for most cases in 8337fe8bcea2cba3c35c54b279f7145bccbc5783.